Commands/Commands/uservoice.js

```diff
@@ -241,6 +241,79 @@ commands.comment = {
   }
 }
 
+commands.info = {
+	modOnly: false,
+	adminOnly: false,
+	fn: function(bot, msg, suffix, uvClient, cBack) {
+		msg.channel.sendTyping()
+		let mentions = msg.mentions()
+		let userid = null
+		if (mentions[0] != null) userid = mentions[0].id
+		else if (suffix.match(/(\d{16,18})/) != null) {
+			let nameid = suffix.match(/(\d{16,18})/)
+			userid = nameid[1].id
+		}
+		else if (suffix.match(/(.{2,32}#\d{4})/)) {
+			let namedisc = suffix.match(/(.{2,32})#(\d{4})/)
+			let members = msg.guild.members
+			let selectedMember = null
+			members.forEach(function(member) {
+				if (member.username == namedisc[1] && member.discriminator == namedisc[2]) userid = member.id
+			})
+    }
+		else msg.channel.sendMessage("Invalid usage. Use a mention, ID or Username#Discriminator combination.").then(errMsg => {
+		setTimeout(bot.deleteMessages([msg,errMsg], msg.channel)),config.timeouts.errorMessageDelete
```

Comment:
Yeah. I was only fixing the syntax, didn't really read the code tbh, it's late ok